cmake/SundialsSetupTPLs.cmake

```diff
@@ -78,6 +78,35 @@ if(ENABLE_HYPRE)
   list(APPEND SUNDIALS_TPL_LIST "HYPRE")
 endif()
 
+# ---------------------------------------------------------------
```

Comment:
I think we should move the Kokkos blocks here to a `cmake/tpls/SundialsKokkos.cmake` file instead (even though its short) for consistency.